Update exampe metric aggregation query (#9202)
opensearch-project/documentation-website: _search-plugins/star-tree-index.md

- The calendar intervals in a star-tree mapping configuration can use either the request's calendar field or a field of lower granularity than the request field. For example, if an aggregation uses the `month` field, the star-tree search can still use lower-granularity fields such as `day`.
- A metric sub-aggregation must be part of the aggregation request.

> The following example gets the sum of all the values in the `size` field with a range query, aggregated for each calendar month:

Author: @kolchfa-aws: how does this sound?

```suggestion
The following example gets a range of status codes between `200` and `400`, then aggregates the sum of all the values for each calendar month:
```

Reviewer: This is good. What does the `size` field contain? Is it the request size? Response size? It would be nice to specify this. In terms of the intro sentence, how about:
```suggestion
The following example first filters logs to include only those with status codes between `200` and `400`. It then aggregates the filtered logs by calendar month and calculates the total sum of values in the `size` field for each month:
```

Author: @kolchfa-aws: Adjusted.

Reviewer: Thank you but could you clarify what is meant to be kept in the `size` field? I think it's essential for users' understanding of this example.

Author: `size: 0` indicates that the response will not return individual document hits and instead only return the aggregation results. Is that what you're referring to, or are you referring to the `"field": "size"` in the aggregation itself?

- The calendar intervals in a star-tree mapping configuration can use either the request's calendar field or a field of lower granularity than the request field. For example, if an aggregation uses the `month` field, the star-tree search can still use lower-granularity fields such as `day`.
- A metric sub-aggregation must be part of the aggregation request.

> The following example filters logs to include only those with status codes between `200` and `400` and sets the `size` of the response to `0`, so that only aggregated results are returned in the response. It then aggregates the filtered logs by calendar month and calculates the total `size`, the total sum of successful and unsuccessful requests, for each month:

Reviewer: ```suggestion
The following example filters logs to include only those with status codes between `200` and `400` and sets the `size` of the response to `0`, so that only aggregated results are returned. It then aggregates the filtered logs by calendar month and calculates the total `size` of the requests for each month:
```

The following example gets the sum of all the values in the `size` field with a range query, aggregated for each calendar month, for all error logs containing `method:get`:

```json

> POST /logs/_search

Author: @kkhatua: It was a mistake when replacing the query, should be fixed now.
